### ISSUE TITLE
Build failure on Linux.

### ISSUE THREAD

**Issue by rageworx** [rageworx/libtinydicom#6]
There are many errors occurs by using cstdint types.

**Comment by rageworx** [comment]
Error reason,
```
In file included from /usr/include/c++/5/cstdint:35:0,
                 from src/dicomtagreader.cpp:4:
/usr/include/c++/5/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support \
```

**Comment by rageworx** [comment]
Solved cstdint errors by including -std=c++11 in Makefile for gcc.

**Comment by rageworx** [comment]
Bugs fixed at commit of b3bb0a3..47fe401.
